app/src/main/java/com/keylesspalace/tusky/components/compose/MediaUploader.kt

```diff
@@ -72,61 +77,40 @@ class MediaUploader @Inject constructor(
     private val context: Context,
     private val mastodonApi: MastodonApi
 ) {
-    fun uploadMedia(media: QueuedMedia): Observable<UploadEvent> {
-        return Observable
-            .fromCallable {
-                if (shouldResizeMedia(media)) {
-                    downsize(media)
-                } else media
+
+    @OptIn(ExperimentalCoroutinesApi::class)
+    fun uploadMedia(media: QueuedMedia): Flow<UploadEvent> {
```

Comment:
I don't think we need a `Flow` here, do we ever emit more than one event? I don't think so